TACC/Core-Styles
# feat: better manage accent colors, darken three light purples

## Overview

Accent colors were set within multiple folder at multiple levels.

This changes them to:
- all be in one folder, no nesting
- load purple accent colors from one place

<details><summary>The caveat of the change is that the three lightest colors become slightly darker (for accessibility).</summary>

**Before**
```css
  --global-color-accent--xxx-light: #faf4ff;
  --global-color-accent--xx-light: #f1ebfd;
  --global-color-accent--x-light: #e3d7fd;
```

**After**
```css
  --global-color-accent--xxx-light: #f1ebfd;
  --global-color-accent--xx-light: #e3d7fd;
  --global-color-accent--x-light: #d5bfff;
```

The purple colors were prepared in #149 (v2.6.2) but unused, specifically because the colors were changed. The change is to improve **both** legibility/accessibility against light backgrounds **and** consistency with blue accent color deltas between tints and shades of blue.

</details>

## Related

- [TUP-463](https://jira.tacc.utexas.edu/browse/TUP-463)
   _([TACC/tup-ui CSS](https://github.com/TACC/tup-ui/tree/v1.0.5/apps/tup-cms/src/taccsite_custom/tup_cms/static/tup_cms/css) builds atop these colors. Simplifying them will make easier the integration of those into Core-CMS and Core-Portal and Core-Styles.)_

## Changes

- **moved** `generics/**/color…` to `settings/color…`
- **changed** cms accent colors to load demo accent colors
- **moved** selectors from blue and purple accent color stylesheets to internal client* stylesheets
- **changed** docs and portal colors to use purple accent color stylesheets
- **renamed** color stylesheets to be (i hope) clearer

<sup>* I.e. demo, cms, docs, portal.</sup>

## Testing

1. Verify accent hues are not different than #175.
    - where had blue accent colors should still have blue accent colors
    - where had purple accent colors should still have purple accent colors
2. Verify three purple colors are lighter than #175.
    - `--global-color-accent--xxx-light`
    - `--global-color-accent--xx-light`
    - `--global-color-accent--x-light`

## UI

### 1. Hues Match #175

https://github.com/TACC/Core-Styles/assets/62723358/1350800d-69b5-4fd6-a72a-5e35a59942fa

### 2. The Lightest Purples Are Slightly Darker

| | after ↔ before |
| - | - | 
| d<br />o<br />c<br />s | ![docs colors](https://github.com/TACC/Core-Styles/assets/62723358/19f4c14f-272c-4aa0-a55f-ff638bb6e4d9)
| p<br />o<br />r<br />t<br />a<br />l | ![portal colors](https://github.com/TACC/Core-Styles/assets/62723358/c4904343-b9f4-4c68-ac10-ffe519a2f614)
